conda_smithy/lint_recipe.py

```diff
@@ -11,16 +11,28 @@
 import jinja2
 import ruamel.yaml
 
-from conda_build.metadata import ensure_valid_license_family
+from conda_build.metadata import ensure_valid_license_family, FIELDS
 
 
+FIELDS['extra'] = ['recipe-maintainers']
+
 EXPECTED_SECTION_ORDER = ['package', 'source', 'build', 'requirements',
                           'test', 'app', 'about', 'extra']
 
 REQUIREMENTS_ORDER = ['build', 'run']
 
 TEST_KEYS = {'imports', 'commands'}
 
+EXPECTED_SUBSECTION_HEADINGS = {'package': ['name', 'version', ],
+                                'source': ['fn', 'url', 'sha256', ],
+                                'build': ['number', 'script'],
```

Comment:
Needs `noarch, features, track_features, detect_binary_files_with_prefix, binary_relocation, entry_points, rpaths`, etc. I don't know if there's anything I missed.

Reply:
Is there a place where these are centrally located?